shared/nav.native.js

```diff
@@ -125,19 +130,18 @@ const forAndroid = ({hideNav, shim, tabBar}) => (
 
 function MainNavStack(props: Props) {
   const screens = props.routeStack
-  const shim = (
-    <Box style={flexOne}>
-      <CardStackShim
-        key={props.routeSelected}
-        stack={screens}
-        renderRoute={renderStackRoute}
-        onNavigateBack={props.navigateUp}
-      />
-      {![chatTab].includes(props.routeSelected) &&
-        <Offline reachability={props.reachability} appFocused={true} />}
-      <GlobalError />
-    </Box>
-  )
+  const shim = [
```

Comment:
this is always in a box so just return an array. with react fiber this is the real fix for a lot of these issues cause you can return and array from a render and you don't have to always return a single item